BUILD/familiars/item.dat

```diff
@@ -22,6 +24,8 @@ Reagnimated Gnome	!path:Heavy Rains
 XO Skeleton
 # Fairy that drops bacon with no limit. 1 per combat
 Intergnat
+# Fairyeverything
+Jill-of-All-Trades
```

Comment:
LoL. This will stop all the griping in Discord about how the Hobo Sheep is autoscend's favourite familiar 😁